netjsonconfig/backends/openwrt/converters/radios.py

```diff
@@ -92,12 +115,25 @@ def __netjson_radio(self, radio):
         radio['channel_width'] = self.__netjson_channel_width(radio)
         return radio
 
+    def __netjson_hwmode(self, radio):
+        if self.dsa:
+            band = radio.pop('band')
+            if band == '2g':
+                radio['hwmode'] = '11g'
+            if band == '5g':
+                radio['hwmode'] = '11a'
+            if band == '6g':
+                radio['hwmode'] = '11ax'
+            if band == '60g':
+                radio['hwmode'] = '11ad'
```

Comment:
From what source of docs are you getting that `11ax` and `11ad`  are valid values here?

> Specifies the hardware mode, possible values are 11b for 2.4 GHz (used for legacy 802.11b only), 11g for 2.4 GHz (used for 802.11b, 802.11g and 802.11n) and 11a for 5 GHz (used for 802.11a, 802.11n and 802.11ac). Note that 11ng, 11na, 11n and 11ac are invalid options, this setting should largely be seen as controlling the frequency band. (DEPRECATED since 21.02.2, replaced with band)

Reply:
This will not present in native OpenWrt configuration, this will be used only in netjson object. Should we show band instead of this in netjson?

Reply:
hwmode is openwrt specific (on openwrt < 21), so if it's not needed I don't think there's a good reason to have this here.

We could add `band` to the netjson schema if it makes sense to do so, what idea do you have in mind?

Reply:
We can add `band` to netjson schema, but we still have to parse or get `hwmode` from band to  predict the protocol (`__netjson_protocol`):
https://github.com/openwisp/netjsonconfig/blob/bdbf4d9afc3de40c4b02e81a76791a43cb4a2a35/netjsonconfig/backends/openwrt/converters/radios.py#L95-L107